tao_compiler/mlir/xla/ral/context/common_context_impl_mkldnn.cc

```diff
@@ -176,6 +176,20 @@ struct MkldnnConvState : public Context::Resource {
 };
 
 #if defined(TAO_AARCH64)
+
+// Returns true is ACL AMP is enabled (`DISC_CPU_ACL_USE_AMP`)
+bool useAclAMP() {
+  static bool flag = []() {
+    const char* env = getenv("DISC_CPU_ACL_USE_AMP");
```

Comment:
Nit: tensorflow::ReadBoolFromEnvVar

Reply:
We do not want to introduce the TF dependency in RAL whenever possible.

Reply:
How about implement something like the function utility in RAL?

Reply:
OK, I personally prefer to do this in a separate PR, including changing other places of using env flags. I'll send the PR later.

Reply:
Thanks!